Observer/AdyenCcDataAssignObserver.php

```diff
@@ -46,6 +46,7 @@ class AdyenCcDataAssignObserver extends AbstractDataAssignObserver
     const SCREEN_HEIGHT = 'screen_height';
     const TIMEZONE_OFFSET = 'timezone_offset';
     const LANGUAGE = 'language';
+    const GUEST_EMAIL = 'guestEmail';
```

Comment:
let's refactor the names in a next PR - guestEmail -> guest_email

Reply:
agree, please create a ticket for this.